### PR TITLE
DOC: fix simple typo, valudators -> validators

### PR DESCRIPTION
**Pull request by timgates42**
There is a small typo in ibis/expr/rules.py.

Should read `validators` rather than `valudators`.

Semi-automated pull request generated by
https://github.com/timgates42/meticulous/blob/master/docs/NOTE.md

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
Thanks for the fix @timgates42. Looks like the CI is broken at the moment, and we may wait until it's fixed to merge this. But all good from your side.

**Comment by jreback** [comment]
thanks @timgates42

**Pull request merged**
